packages/core/stories/pill/selectable-pill.stories.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  FormField,
+  FormFieldHelperText,
+  FormFieldLabel,
+  Pill,
+  PillGroup,
+} from "@salt-ds/core";
+import { FavoriteIcon, UserBadgeIcon } from "@salt-ds/icons";
+import type { Meta, StoryFn } from "@storybook/react-vite";
+import { useState } from "react";
+
+export default {
+  title: "Core/Pill/Selectable",
+  component: Pill,
+} as Meta<typeof Pill>;
+
+export const Default: StoryFn<typeof PillGroup> = (args) => {
+  return (
+    <PillGroup aria-label="Group label" {...args}>
+      <Pill value="one">Pill 1</Pill>
+      <Pill value="two">Pill 2</Pill>
+      <Pill value="three">Pill 3</Pill>
+    </PillGroup>
+  );
+};
+
+export const WithWrap = () => {
+  return (
+    <PillGroup aria-label="Group label" style={{ maxWidth: 250 }}>
+      {Array.from({ length: 20 }).map((_, index) => (
+        // biome-ignore lint/suspicious/noArrayIndexKey: Mock data
+        <Pill key={index} value={`pill-${index}`}>
+          Pill {index + 1}
+        </Pill>
+      ))}
+    </PillGroup>
+  );
+};
+
+export const WithIcon = () => {
+  return (
+    <PillGroup aria-label="Group label">
+      <Pill value="one">
+        <UserBadgeIcon />
```

Comment:
These icons should have aria-hidden following our icon best practices https://www.saltdesignsystem.com/salt/components/icon/accessibility#best-practices